vignettes/non-english.Rmd

```diff
@@ -0,0 +1,79 @@
+---
+title: "Non-English sites"
+output: rmarkdown::html_vignette
+description: > 
+  If your documentation is written in a language other than English, you can 
+  generate text in your language using the `lang` field. You will also learn
```

Comment:
- "generate text" should be "use template pieces" or so.
- The second sentence refers to the vignette whereas the first one does not. Either make both of them about the vignette "in this vignette you will learn..." or rephrase the second one to "You can provide your own translations if your language isn't currently supported pkgdown; we'll show you how". :thinking: